vnext/local-cli/generator-windows/templates/cs/proj/MyApp.csproj

```diff
@@ -154,25 +153,33 @@
     </PackageReference>
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="$(ReactNativeWindowsDir)\Microsoft.ReactNative\Microsoft.ReactNative.vcxproj">
-      <Project>{f7d32bd0-2749-483e-9a0d-1635ef7e3136}</Project>
-      <Name>Microsoft.ReactNative</Name>
-    </ProjectReference>
-    <PackageReference Include="Microsoft.UI.Xaml">
-      <Version>2.3.191129002</Version>
-    </PackageReference>
+    <Reference Include="Microsoft.UI.Xaml, Version=255.255.255.255, Culture=neutral, PublicKeyToken=null, ContentType=WindowsRuntime">
+      <HintPath>..\packages\Microsoft.UI.Xaml.2.3.191129002\lib\uap10.0\Microsoft.UI.Xaml.winmd</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="Microsoft.UI.Xaml.Design, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null">
+      <HintPath>..\packages\Microsoft.UI.Xaml.2.3.191129002\lib\uap10.0\Design\Microsoft.UI.Xaml.Design.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
```

Comment:
we shouldn't reference the WinMD directly in CS projects, we should just PackageReference the package.
Put the PackageReference in the targets file

Reply:
this is inserted by nuget and must be here in order for nuget to support updating it

Reply:
you are right but that's because you're making the csproj be a packages.config style project. 
(`nuget update` is only for packages.config style projects):

![image](https://user-images.githubusercontent.com/22989529/81267236-fac48300-8ffa-11ea-9375-33736250fbd1.png)

The right way for .net projects to use nuget packages is to use PackageReference instead. packages.config isn't being developed and is a much clunkier way - e.g. it requires you to list all the things you want to link into your app. In this case you're linking in the Designer which apps do not need/want, this is only used by VS itself.

Reply:
:( I thought that UWP added support to vcxproj for packagereference.

per: [https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/consume-packages/Package-References-in-Project-Files.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/consume-packages/Package-References-in-Project-Files.md)

This is tracked by vs in [this community request](https://developercommunity.visualstudio.com/idea/351636/use-packagereference-in-vcxproj.html)